org.easyrpg.player.yml

```diff
@@ -0,0 +1,84 @@
+app-id: org.easyrpg.player
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: easyrpg-player
+rename-desktop-file: easyrpg-player.desktop
+rename-appdata-file: easyrpg-player.metainfo.xml
+rename-icon: easyrpg-player
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --filesystem=home # user can store games anywhere
+  - --device=all # needed for gamepad access
+modules:
+  # external libraries
+  - shared-modules/linux-audio/fluidsynth2.json
+  - shared-modules/SDL2/SDL2-with-libdecor.json
+  - name: libfmt
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DFMT_TEST=OFF
+      - -DBUILD_SHARED_LIBS=TRUE
+    sources:
+      - type: git
+        url: https://github.com/fmtlib/fmt.git
+        tag: '9.1.0'
+        commit: a33701196adfad74917046096bf5a2aa0ab0bb50
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /lib/cmake
+      - '*.so'
```

Comment:
Put the common ones in a top level cleanup. Won't need repeating for every module.